### PR TITLE
Add guide for livingdocs assistants

### PR DESCRIPTION
**Pull request by dfreier**
@benib @fuenkchen Happy to align this further with the upcoming release communication

### CODE REVIEW

File: content/guides/editor/assistants/index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: AI Assistants
+description: Take the editor experience to the next level and hook into Livingdocs AI Assistants
```
Comment (benib) [inline content/guides/editor/assistants/index.md:3]: I suggest to name it `Livingdocs Assistants`. They can, but don't have to use AI. I would like to keep some of the Buzzword/Magic out of the game here. It's just `Assistants`, some piece of logic that was deliberately put there by a human to make the live of Livingdocs users easier, not some kind of AI magic.

**Comment by benib** [comment]
Thank you so much for writing this up @dfreier. I'll work on some wording for the cycle demo and release notes tomorrow afternoon. And I'll definitely take inspiration from your writeup here. Then we can align in later this week, sounds good?

**Comment by peyerluk** [comment]
@dfreier Is this ready to be merged? And have you planned a reference documentation for assistants as well in the Server Extensions section?

**Comment by benib** [comment]
> @benib I was just about to update the code example with old `iconUrl` 😄 I'll open another PR

ups, sorry for that 🙈 and thank you!

**Pull request merged**